recipes/trufflehog/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "trufflehog" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/truffleHog-{{ version }}.tar.gz
+  sha256: 7f0d09c8cda2a90ae42f81405e5944a3325d90a5842d2dde014471e4d65c71e4
+
+build:
+  entry_points:
+    - trufflehog = truffleHog.truffleHog:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.11
+    - pip
+  run:
+    - python >=3.7,<3.11
+    - gitpython >=3.0.5,<3.1.0
+    - trufflehogregexes ==0.0.7
+
+test:
+  imports:
+    - truffleHog
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dxa4481/truffleHog
+  summary: Searches through git repositories for high entropy strings, digging deep into commit history.
+  license: gnuplot
```

Comment:
I'm not sure that is a valid license. It looks like a `AGPL-3.0-or-later`.

Reply:
BTW, this is the gnuplot one for comparison: https://spdx.org/licenses/gnuplot.html